Gemfile

```diff
@@ -3,5 +3,5 @@ git_source(:github) { |repo| "https://github.com/#{repo}.git" }
 
 gemspec
 
-gem 'workarea-api', github: 'workarea-commerce/workarea-api'
+gem 'workarea-api', github: 'workarea-commerce/workarea-api', branch: 'master'
```

Comment:
this is just so bundler doesn't yell at me. i have a local override set up for workarea-api, and it only works if you specify the branch. not sure why this limitation exists, but it shouldn't affect the codebase.

Reply:
let's not commit this if you just need it locally.